src/convertToCase/validate.js

```diff
@@ -0,0 +1,23 @@
+function validate(text, textCase) {
+  const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!text.length) {
+    // eslint-disable-next-line max-len
+    errors.push({ message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
```

Comment:
Let's create an object with error messages outside the validate function and use it here